tests/pds/dataservice_test.go

```diff
@@ -25,6 +26,29 @@ const (
 	defaultRebootTimeRange       = 5 * time.Minute
 )
 
+var _ = Describe("{CollectLogs}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("DeletePDSPods", "delete pds pods and validate if its coming back online and dataserices are not affected", nil, 0)
+	})
+
+	It("Collect logs", func() {
+		Step("collect container logs", func() {
+			isDeploymentsDeleted = false
+			err = pdslib.CollectPodLogsandValidateWorkloads()
+			err = errors.New("an error message")
+			Expect(err).ShouldNot(HaveOccurred())
+			//log.FailOnError(err, "Failed while collecting pod logs")
+		})
+	})
+	JustAfterEach(func() {
+		if !isDeploymentsDeleted {
+			Step("Delete created deployments")
```

Comment:
Looks like we saying we going to delete created deployment, but not doing anything here, doesn't seem right